src/game/Game.java

```diff
@@ -1,7 +1,16 @@
+package game;
 import java.util.Date;
 import java.util.Random;
 import java.util.Scanner;
 
+import model.Board;
```

Comment:
Can u not just import all of the `model` package at once?

Reply:
im not sure, that was automatic